src/web/components/KeyPairs/KeyPairDialog.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
```

Comment:
nit: for the new story could we use CSF 3.0

here is the doc: https://storybook.js.org/docs/react/writing-stories/introduction

Reply:
I changed it.